weblab/entities/migrations/0016_delete_fitting_spec_protocols.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.16 on 2020-11-25 15:02
+
+from django.db import migrations
+
+def delete_protocols(apps, schema_editor):
+
+    Protocol = apps.get_model('entities', 'ProtocolEntity')
+    for protocol in Protocol.objects.all():
+        if protocol.is_fitting_spec:
+            protocol.delete();
+
```

Comment:
I have not tested this yet also do I need to put in the reverse migration

Reply:
We don't need the reverse migration - there's not really anything we can do for it given we delete data!

One suggestion on the above: you can use `Protocol.objects.filter(is_fitting_spec=True)` to do the filtering in the DB rather than in Python.